Server/JokesOnYou.Web.Api/Services/Interfaces/ITokenService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using JokesOnYou.Web.Api.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JokesOnYou.Web.Api.Services.Interfaces
+{
+    interface ITokenService
```

Comment:
Public?

Reply:
correct this is me being dumb I thought Interfaces were public by default :P